plugins/plugin-sass/plugin.js

```diff
@@ -0,0 +1,76 @@
+const execa = require('execa');
+const fs = require('fs');
+const path = require('path');
+
+const IMPORT_REGEX = /\@(use|import)\s*['"](.*?)['"]/g;
+
+function scanSassImports(fileContents, filePath) {
+  // TODO: Replace with matchAll once Node v10 is out of TLS.
+  // const allMatches = [...result.matchAll(new RegExp(HTML_JS_REGEX))];
+  const allMatches = [];
+  let match;
+  const regex = new RegExp(IMPORT_REGEX);
+  while ((match = regex.exec(fileContents))) {
+    allMatches.push(match);
+  }
+
+  return allMatches
+    .map((match) => match[2])
+    .filter((s) => s.trim())
+    .map((s) => {
+      if (!path.extname(s)) {
+        s += '.scss';
```

Comment:
Might need to try and resolve `.sass` files as a 2nd try?

Reply:
think I came up with a good shortcut, lmk what you think